ngraph/test/backend/deformable_convolution.in.cpp

```diff
@@ -0,0 +1,2024 @@
+                                    5.0f, 6.0f, 7.0f, 8.0f,
+                                    9.0f, 10.0f, 11.0f, 12.0f,
+                                    13.0f, 14.0f, 15.0f, 16.0f};
+
+    const Shape filter_shape{1, 1, 2, 2};
+    const std::vector<float> filter{1.0f, 2.0f,
+                                    -1.0f, -2.0f};
+
+    const Shape offsets_shape{1, 8, 3, 3};
+    const std::vector<float> offsets{
+                                    // window 1 (Y=0, X=0) -> Y coordinate
+                                     1.1f, 1.1f, 1.1f, // out1 .. out 3
+                                     1.1f, 1.1f, 1.1f, // out4 .. out 6
+                                     1.1f, 1.1f, 1.1f, // out7 .. out 9
+                                     // window 1 (Y=0, X=0) -> X coordinate
+                                     1.1f, 1.1f, 1.1f, // out1 .. out 3
+                                     1.1f, 1.1f, 1.1f, // out4 .. out 6
+                                     1.1f, 1.1f, 1.1f, // out7 .. out 9
+                                     // window 2
+                                     1.1f, 1.1f, 1.1f,
+                                     1.1f, 1.1f, 1.1f,
+                                     1.1f, 1.1f, 1.1f,
+                                     // window 2
+                                     1.1f, 1.1f, 1.1f,
+                                     1.1f, 1.1f, 1.1f,
+                                     1.1f, 1.1f, 1.1f,
+                                     // window 3
+                                     1.1f, 1.1f, 1.1f,
+                                     1.1f, 1.1f, 1.1f,
+                                     1.1f, 1.1f, 1.1f,
+                                     // w1indow 3
+                                     1.1f, 1.1f, 1.1f,
+                                     1.1f, 1.1f, 1.1f,
+                                     1.1f, 1.1f, 1.1f,
+                                     // window 4
+                                     1.1f, 1.1f, 1.1f,
+                                     1.1f, 1.1f, 1.1f,
+                                     1.1f, 1.1f, 1.1f,
+                                     // window 4
+                                     1.1f, 1.1f, 1.1f,
+                                     1.1f, 1.1f, 1.1f,
+                                     1.1f, 1.1f, 1.1f,
+                                    };
+
+    const Shape outputs_shape{1, 1, 3, 3};
+    const std::vector<float> outputs{-11.999998f, -11.999999f, -4.000000f,
+                                     -10.799999f, -10.800001f, -3.600004f,
+                                     44.300000f, 47.100000f, 16.000000f};
+
+    const size_t tolerance_bits = 6;
+    DeformableConvolutionTest(inputs, inputs_shape, offsets, offsets_shape, filter,
+                              filter_shape, outputs, outputs_shape, strides, padding,
+                              dilations, group, deformable_group, tolerance_bits);
+}
+
+// TODO: group & deformable_group attributes (real offsets)
```

Comment:
TODO: add more test of "group & deformable_group attributes (real offsets)" type

Reply:
Done in [here](https://github.com/openvinotoolkit/openvino/pull/4312/commits/c57481a30be7b9691d32fd94b9a4f9801d93059b).